Makefile

```diff
@@ -28,6 +30,7 @@ DUCKDB_COMMON_BUILD_FLAGS := BUILD_SHELL=0 BUILD_UNITTESTS=0 DUCKDB_PLATFORM=any
 .PHONY: deps.darwin.amd64
 deps.darwin.amd64: duckdb
 	if [ "$(shell uname -s | tr '[:upper:]' '[:lower:]')" != "darwin" ]; then echo "Error: must run build on darwin"; false; fi
+	mkdir -p deps/darwin_amd64
```

Comment:
Why is this `mkdir -p deps/respective_build` necessary in all the builds?

Reply:
I don't know, I think it was mostly for consistency? It has been a while, I can remove it if you want

Reply:
Oh I needed it because there didn't exist a directory yet. The usecase may be small, but I think in general it is nicer to not assume a directory already exists, even if we know it does. I can remove it if you want to,

Reply:
That's a fair point, let's keep it 👍